### PR TITLE
 BOIL-1: User should be able to register using email and password

### PR DESCRIPTION
**Pull request by Shubhankardubey**
## Description
- user registration is done.
- validations done for empty input values, same email address, password & cnf_password mismatch    
- locales are loaded 
- config files updated 
- register button added at landing page
- app.json upda

## Screenshot
screencapture-boilerplate-webapp-stagin-pr-1-herokuapp-register-2019-01-16-12_32_57

### Manual test cases run
- Validations are applied for missing input values, invalid `email`, `password` & `cnf_password` not matches.
- No account is created for same `email`
- If all goes well it will console **done**

### Browsers tested
- [ ] Chrome
- [ ] Safari
- [ ] Internet Explorer
- [ ] Firefox

## Platform dependencies
Does this depend on or impact
- [ ] iOS
- [ ] Android
- [ ] API

### CODE REVIEW

File: src/assets/i18n/en.json [hunk 1]
```diff
@@ -1,3 +1,217 @@
 {
-  "HELLO_WORLD": "Hello World"
+  "FORM_HEADINGS": {
```
Comment (jjalan) [inline src/assets/i18n/en.json]: There are lot of strings in this file which are not being used in this project. Please remove them.

File: src/app/modules/accounts/pages/register/register.component.html [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+              <div class="row mt-2" *ngIf="accountCreationViewState.isFinishedWithSuccess">
+                <span class="col-md-12 text-center text-info">{{'FORM_MESSAGES.USR_CREATE_ACC_OK' | translate}}</span>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="row mt-4">
+        <div class="col-md-12 form-group text-center">
+          &copy; 2018 AGREGAT
```
Comment (jjalan) [inline src/app/modules/accounts/pages/register/register.component.html]: Company name should YoAccess Technology Solutions Pvt. Ltd.

File: src/app/pages/about-us/about-us.component.ts [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import {Component} from '@angular/core';
 
+
```
Comment (jjalan) [inline src/app/pages/about-us/about-us.component.ts]: No change, only new lines.

Please revert changes in this file.

Comment (Shubhankardubey) [reply src/app/pages/about-us/about-us.component.ts]: Resolved

File: src/app/modules/accounts/pages/register/register.component.html (hunk 2 above)
Comment (jjalan) [reply src/app/modules/accounts/pages/register/register.component.html]: Actually, we should remove it as there is already a footer that contains the copyright information.

File: src/app/pages/about-us/about-us.component.ts [hunk 4]
```diff
@@ -8,4 +8,5 @@ import {Component} from '@angular/core';
 })
 
 export class AboutUsComponent {
+
```
Comment (jjalan) [inline src/app/pages/about-us/about-us.component.ts]: Extra line, please revert.

File: src/app/modules/accounts/pages/register/register.component.html (hunk 2 above)
Comment (Shubhankardubey) [reply src/app/modules/accounts/pages/register/register.component.html]: resolved

File: src/assets/i18n/en.json (hunk 1 above)
Comment (Shubhankardubey) [reply src/assets/i18n/en.json]: resolved

File: src/app/pages/about-us/about-us.component.ts (hunk 4 above)
Comment (Shubhankardubey) [reply src/app/pages/about-us/about-us.component.ts]: resolved

**Pull request merged**